components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint/src/main/webapp/password-recovery.jsp

```diff
@@ -135,8 +135,7 @@
                                         </td>
                                         <td>&nbsp;&nbsp;</td>
                                         <td>
-                                            <button id="recoveryCancel"
-                                                    class="wr-btn grey-bg col-xs-12 col-md-12 col-lg-12 uppercase font-extra-large"
+                                            <button type="button" id="recoveryCancel" class="wr-btn grey-bg col-xs-12 col-md-12 col-lg-12 uppercase font-extra-large"
```

Comment:
Shouldn't this be \<input type="button" /> ?

Reply:
No, button also can have type button